src/internal/m365/collection/exchange/backup.go

```diff
@@ -278,7 +296,7 @@ func populateCollections(
 
 	collections["metadata"] = col
 
-	logger.Ctx(ctx).Infow("produced collections", "count_collections", len(collections))
+	logger.Ctx(ctx).Infow("produced collections", "stats", counter.Values())
```

Comment:
Sanity check, since this is not a local count bus, we'll end up logging stats from parent funcs too right?

Reply:
Ah, yeh.  I've swapped the counter passed into m365 `ProduceBackupCollections()` from operations to be a local instance, so that it more cleanly encapsulates the producer stats relative to other ops level stats.